dev/import-beats/packages.go

```diff
@@ -0,0 +1,73 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+
+	"github.com/elastic/package-registry/util"
+)
+
+type packageMap map[string]*util.Package
+
+func (p packageMap) loadFromSource(beatsDir, beatName, packageType string) error {
+	path := filepath.Join(beatsDir, beatName, "module")
+	moduleDirs, err := ioutil.ReadDir(path)
+	if err != nil {
+		return errors.Wrapf(err, "cannot read directory '%s'", path)
+	}
+
+	for _, moduleDir := range moduleDirs {
+		if !moduleDir.IsDir() {
+			continue
+		}
+
+		log.Printf("Visit '%s:%s'\n", beatName, moduleDir.Name())
+
+		_, ok := p[moduleDir.Name()]
```

Comment:
Perhaps worth a comment that this is needed because in packages logs and metrics modules are combined.

Reply:
Comment left.